planning/behavior_path_planner/include/behavior_path_planner/behavior_tree_manager.hpp

```diff
@@ -49,6 +50,9 @@ class BehaviorTreeManager
 
   BehaviorModuleOutput run(const std::shared_ptr<PlannerData> & data);
   std::vector<std::shared_ptr<SceneModuleStatus>> getModulesStatus();
+  std::shared_ptr<SceneModuleVisitor> get_all_debug_data();
```

Comment:
please use the camelCase style as the other functions. Since ROS2 suggests using the snake_case, we will change all functions at the same time.

Reply:
Done. Fixed in 57c1ed16af196a0dfcd745d11a0ce6065fccac54